Constraints/MessagesData.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  MessagesData.swift
+//  Constraints
+//
+//  Created by Angela Lee on 16/08/2022.
+//
+
+import Foundation
+
+struct MessagesData: Codable {
```

Comment:
This struct name should be written in singular, "MessageData", since your are modeling one message, and later you are going to have several instances of it. You need to apply the same naming rule as Classes.

Reply:
Corrected